### DISCUSSION TITLE
Media deletion label disambiguation

### DISCUSSION THREAD

**Discussion by hdavidkim** [jellyfin/jellyfin-web#5277] [Ideas]
Hello, I would like to take on [this feature request](https://features.jellyfin.org/posts/2573/delete-series-confirmation-to-be-different-to-delete-episode) as it seems pretty straightforward and would indeed be a good QoL improvement. I'll be checking if the item type is a series or episode and will change the string to indicate as such. Looking over how item types are defined, I may follow up with an increased scope feature so that each different item are clearly given a specific deletion label.

Am I correct in assuming that I should just come up with translation keys as needed and then go into the [translator app](https://translate.jellyfin.org/translate/jellyfin/jellyfin-web/en/) to add in the entries for english with other languages soon following suit (by someone else since I don't know those languages)?

This will be my first code change for jellyfin so please give me whatever tips you feel are appropriate.

Thanks!

**Comment by hdavidkim** [comment]
also, am i correct in assuming that the docker file I use to spin up a test app is [Dockerfile.docker](https://github.com/jellyfin/jellyfin-web/blob/master/deployment/Dockerfile.docker) or is that just for production deployment? I'm pretty new to docker so I'm a bit confused on how to launch and test the webapp using it

**Comment by dmitrylyzo** [comment]
There is a PR already.
https://github.com/jellyfin/jellyfin-web/pull/5232

**Comment by hdavidkim** [comment]
oh, that's great. added my thoughts on the PR there. Could we get the [feature request](https://features.jellyfin.org/posts/2573/delete-series-confirmation-to-be-different-to-delete-episode) marked as "started"?

also, am i able to approve that other PR or is it policy to wait for someone with more jellyfin dev experience to give a final review?

**Reply by nielsvanvelzen** [reply]
Pull requests in the web repository need at least 1 approval from a Jellyfin team member in the web team. Other contributors can help out, but it always needs 1 team member.
